docs/non-normative/status-metrics.md

```diff
@@ -0,0 +1,95 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Status Metrics
+aliases: [status-metrics]
+--->
+
+# State Metrics
```

Comment:
could you please add a link to it from https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/how-to-define-semantic-conventions.md#defining-metrics ?